src/app.js

```diff
@@ -44,6 +48,11 @@ class Messenger extends EventEmitter {
     }
     this.options.emitGreetings = !!emitGreetings;
 
+    router.init(this);
+    dispatcher.on('session.changed', ({session}) => {
```

Comment:
Curious on your thoughts, @crccheck & @stripethree. My thought is to allow an object to be passed and only lay claim to `session` as a property so you could emit other things on `session.changed`. Not sure if it's needed or not so I'm kinda on the fence about this verse `… (session) => {`.